tap_pendo/streams.py

```diff
@@ -674,7 +692,7 @@ def get_body(self, period, first):
 class PollEvents(Stream):
     replication_method = "INCREMENTAL"
     name = "poll_events"
-    key_properties = ['visitor_id', 'account_id', 'server_name', 'remote_ip']
+    key_properties = []
```

Comment:
Hello, thank you for the contribution. Could you provide us with more information about the reason for this change to `key_properties = []`, for both the `PollEvents` and `GuideEvents` streams, and how you tested them?

Reply:
Hi! @leslievandemark  , I followed the documentation on ungrouped events https://developers.pendo.io/docs/?bash#events-ungrouped that both PollEvents and GuideEvents below to. For instance in GuideEvents documentation, the first bullet says:

```
Returns one event row for every guide event that occurs. This is not a summary or bucket structure like previous examples.
```

So there is no aggregation involved, so there shouldn't be unique constraints on any keys.

I tested them running the tap, although I didn't have guides or polls to retrieve in the first place